contracts/eosio.bios/include/eosio.bios/eosio.bios.hpp

```diff
@@ -190,6 +220,18 @@ namespace eosiobios {
          [[eosio::action]]
          void onerror( ignore<uint128_t> sender_id, ignore<std::vector<char>> sent_trx );
 
+         /**
+          * Set a new list of finalizer policy.
+          *
+          * @details Set a new list of active finalizer policy, by proposing a finalizer policy, once the block that
```

Comment:
We should not say `once the block that contains the proposal becomes irreversible` since that may not be the rule. We still  have to figure out the appropriate rule.

The comment can just say: `Propose new finalizer policy that, unless superseded by a later finalizer policy, will eventually become the active finalizer policy.`